inst/pages/import.qmd

```diff
@@ -525,6 +525,21 @@ sequencing data, all processed using the same pipeline and reference
 database. For more use examples in R/Bioconductor, see the 
 [MicroBioMap vignette](https://blekhmanlab.github.io/MicroBioMap/articles/overview.html).
 
+### Integrative human microbiome project
+
+Datasets from the second phase of the human microbiome project, also known as
+integrative human microbiome project are made available with `r BiocStyle::Biocpkg("HMP2Data")`[@HMP2data].
+The datasets are offered as `TreeSE` objects. Additional data, *e.g.* cytokines,
+is included when available.
+
+### Microbiome benchmark data
+
+The `r BiocStyle::Biocpkg("MicrobiomeBenchmarkData")` allows access to datasets
+with some groundtruth available[@Gamboa-Tuz2025]. These datasets are compiled for the purpose of
+benchmarking differential abundance methods, but the ground truth may be useful
+for other benchmarking applications. Datasets are offered directily as `TreeSE`
+objects.
```

Comment:
Can you add them also here: https://microbiome.github.io/OMA/docs/devel/pages/miaverse.html#sec-sub-diff-abund

Reply:
Btw, you could try to make the table more pretty. The idea of that table is to list all packages that have TreeSE support.